price-feeder/oracle/provider/osmosis.go

```diff
@@ -175,9 +176,10 @@ func (p OsmosisProvider) GetCandlePrices(pairs ...types.CurrencyPair) (map[strin
 			closeStr := fmt.Sprintf("%f", responseCandle.Close)
 			volumeStr := fmt.Sprintf("%f", responseCandle.Volume)
 			candlePrices = append(candlePrices, types.CandlePrice{
-				Price:     sdk.MustNewDecFromStr(closeStr),
-				Volume:    sdk.MustNewDecFromStr(volumeStr),
-				TimeStamp: responseCandle.Time,
+				Price:  sdk.MustNewDecFromStr(closeStr),
+				Volume: sdk.MustNewDecFromStr(volumeStr),
+				// convert osmosis timestamp seconds -> milliseconds
+				TimeStamp: responseCandle.Time * int64(time.Second/time.Millisecond),
```

Comment:
- We could se this as an const 	`timeSecToMilisec      = int64(time.Second / time.Millisecond)`
and then use the const
```suggestion
				TimeStamp: responseCandle.Time * timeSecToMilisec,
```

Reply:
I used a helper function instead here :^) https://github.com/umee-network/umee/pull/1264/commits/e18ada7138b48ed797645ee1a5222ccb3633c42c